### ISSUE TITLE
nextercism: implement version command 

### ISSUE THREAD

**Issue by kytrinyx** [exercism/cli#412]
_To be merged into the nextercism branch in https://github.com/exercism/cli/pull/410_

Command name: `version` (alias `v`).
File: `cmd/version.go`.

## Current Version

The default behavior of this command is to output the current version.

## Latest Version

When passed the `--latest` (`-l`) flag, it checks whether or not there's a newer version available.

We have logic for this in the `cli` package: https://github.com/exercism/cli/blob/master/cli/cli.go#L66

**Comment by kytrinyx** [comment]
@nywilken You mentioned being interested in getting involved with the new CLI implementation. This issue should be a good first issue to get started with, to help you get your head wrapped around the project.

To work on it, pull down and branch off of the `nextercism` branch on this repository. That's where you would submit the PR back to (you can choose the "base" branch when creating the PR. If you get it wrong—I almost always do—then you can edit the PR to make it go against the correct branch after).

**Comment by kytrinyx** [comment]
Oh, and to set the flag on a command you would stick this in the `init()` for the command:

```
versionCmd.Flags().BoolP("latest", "l", false, "something that describes the flag")
```

**Comment by nywilken** [comment]
Sweet! Thanks for throwing this my way. I will dive in this weekend and reach out if I run into any issues or have questions.

**Comment by kytrinyx** [comment]
Awesome! I'm sticking `good-first-patch` labels on a few other things as well, and then as we dig into it we'll figure out what other pieces would be good to carve off.

**Comment by kytrinyx** [comment]
Did you get a chance to dive in at all? (No pressure, just curious.)

**Comment by nywilken** [comment]
Not yet, I spent some time looking through the PRs you asked for my review to get a sense of how things are currently working.  I planned to attack this tonight and tomorrow.

**Comment by kytrinyx** [comment]
✨ Excellent. Do holler if you run into anything, I plan to be around and working on this both tonight and tomorrow.

**Comment by nywilken** [comment]
Sounds good, thanks.

**Comment by nywilken** [comment]
@kytrinyx any particular reason for using `BoolP` over `BoolVarP`?

```Go
versionCmd.Flags().BoolP("latest", "l", false, "something that describes the flag")
```

Versus 

```Go
versionCmd.Flags().BooVarlP(&checkLatest, "latest", "l", false, "something that describes the flag")
```

**Comment by kytrinyx** [comment]
No, not in this case. With other flags I load them into viper and then a struct, so the var didn't make sense there, but in this case a variable would be better.

**Comment by nywilken** [comment]
Ah okay, that makes senses. Thanks for the quick response. I'll shoukd have a PR for this tomorrow. 

Regarding the messaging, was something formalized within a discussion? Or would something like the following be okay?

```Bash
> exercism v -l
exercism version X.Y.Z
Your CLI [is up to date|does not match the latest version A.B.C].
```

**Comment by kytrinyx** [comment]
We haven't worked out the messaging yet. I'm going to do a full pass on messaging at the end when we see what the scope of it is.

I think it makes sense to call out the upgrade command if a newer version is available.

**Comment by kytrinyx** [comment]
Closed by #432
